extension/landing-zone/variables.tf

```diff
@@ -0,0 +1,328 @@
+        "direction" : "outbound",
+        "name" : "allow-ibm-tcp-80-outbound",
+        "source" : "161.26.0.0/16",
+        "tcp" : {
+          "port_max" : 80,
+          "port_min" : 80
+        }
+      },
+      {
+        "direction" : "outbound",
+        "name" : "allow-ibm-tcp-443-outbound",
+        "source" : "161.26.0.0/16",
+        "tcp" : {
+          "port_max" : 443,
+          "port_min" : 443
+        }
+      }
+    ]
+  }
+}
+
+variable "user_data" {
+  description = "User data to initialize VSI deployment"
+  type        = string
+  default     = null
+}
+
+variable "boot_volume_encryption_key" {
+  description = "CRN of boot volume encryption key"
+  type        = string
+}
+
+variable "existing_kms_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto Services instance in which the key specified in var.boot_volume_encryption_key is coming from."
+  type        = string
+}
+
+variable "skip_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of an IAM authorization policy that permits all Storage Blocks to read the encryption key from the KMS instance. If set to false, pass in a value for the KMS instance in the existing_kms_instance_guid variable."
+  default     = false
+}
+
+variable "vsi_per_subnet" {
+  description = "Number of VSI instances for each subnet"
+  type        = number
+  default     = 1
+}
+
+variable "subnet_names" {
+  description = "Subnets to which the VSI instances should be deployed"
+  type        = list(string)
+  default = [
+    "vpe-zone-1",
+    "vpe-zone-2",
+    "vpe-zone-3"
+  ]
+}
+
+variable "security_group_ids" {
+  description = "IDs of additional security groups to be added to VSI deployment primary interface. A VSI interface can have a maximum of 5 security groups."
+  type        = list(string)
+  default     = []
+
+  validation {
+    error_message = "Security group IDs must be unique."
+    condition     = length(var.security_group_ids) == length(distinct(var.security_group_ids))
+  }
+
+  validation {
+    error_message = "No more than 5 security groups can be added to a VSI deployment."
+    condition     = length(var.security_group_ids) <= 5
+  }
+}
+
+variable "block_storage_volumes" {
+  description = "List describing the block storage volumes that will be attached to each vsi"
+  type = list(
+    object({
+      name           = string
+      profile        = string
+      capacity       = optional(number)
+      iops           = optional(number)
+      encryption_key = optional(string)
+    })
+  )
+  default = []
+
+  validation {
+    error_message = "Each block storage volume must have a unique name."
+    condition     = length(distinct(var.block_storage_volumes[*].name)) == length(var.block_storage_volumes)
+  }
+}
+variable "enable_floating_ip" {
+  description = "Create a floating IP for each virtual server created"
+  type        = bool
+  default     = false
+}
+
+variable "load_balancers" {
```

Comment:
additional load balancer config is coming in https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/pull/536 so ensure thats merged and gets added here before this PR is merged